src/OSPSuite.Core/CoreRegister.cs

```diff
@@ -92,6 +92,9 @@ public override void RegisterInContainer(IContainer container)
          container.Register<ISensitivityAnalysisRunner, SensitivityAnalysisRunner>(LifeStyle.Singleton);
          container.Register<IConfirmationManager, ConfirmationManager>(LifeStyle.Singleton);
 
+         //TODO no idea why I need to register this manually. I don't see where it's being excluded 
+         container.Register<SimulationBuilder, SimulationBuilder>();
```

Comment:
@rwmcintosh I don't understand why I need to do this. Any idea?

Reply:
AssemblyScanner for utility includes this test that filters out classes without any interfaces

```
	private bool cannotRegister(Type type)
	{
		return type.IsInterface || type.IsAbstract || type.IsEnum || type.GetInterfaces().Length == 0;
	}
```